### ISSUE TITLE
Is there a way to create a peripheral from an uuid ?

### ISSUE THREAD

**Issue by mink-lparrouy** [Polidea/RxBluetoothKit#312]
Hi everybody,

I wonder if there is a way to create a Peripheral from an uuid in order to connect to the device without scanning devices ?

I save the uuid in CoreData and when the user kill and open the app again I want to perform an "auto connect".

Thank you for your response

**Comment by dariuszseweryn** [comment]
```
let peripheralsFromUuids: [Peripheral] = manager.retrievePeripherals(withIdentifiers: [UUID])
```

**Comment by maulikTeksun** [comment]
Yes having same issue, Is there any way like default CBCentralManager doing?

 @available(iOS 7.0, *)
   open func retrievePeripherals(withIdentifiers identifiers: [UUID]) -> [CBPeripheral]
